### DISCUSSION TITLE
Possible to slide subpage on top of homepage?

### DISCUSSION THREAD

**Discussion by dbraun-ascendium** [swup/swup#544] [Help] [answered]
I've created a transition where the homepage slides out and the subpage slides in (and vice versa).

However, is it possible to slide the subpage "above" the homepage? In other words, transition to the subpage without first removing the homepage.

Thanks!

**Comment by daun** [comment, marked as answer]
**Update**: the recently released [swup 4](https://swup.js.org/announcements/swup-4/) supports [parallel animations](https://swup.js.org/plugins/parallel-plugin/).

Previous answer for swup 3:

> No, not out of the box. I'll refer your to [a previous discussion about synchronous transitions](https://github.com/swup/swup/discussions/502#discussioncomment-3536642).
